#!/usr/bin/python3
#message for raised exceptions

def raise_exception_msg(message=""):
    raise NameError(message)
